src/core/common/instance.hpp

```diff
@@ -311,10 +313,14 @@ class Instance : public otInstance
     Notifier mNotifier;
     Settings mSettings;
 
-#if !OPENTHREAD_ENABLE_MULTIPLE_INSTANCES
-#if OPENTHREAD_CONFIG_ENABLE_BUILTIN_MBEDTLS
     Crypto::MbedTls mMbedTls;
```

Comment:
With this we will have `mMbedTls` even when `OPENTHREAD_CONFIG_ENABLE_BUILTIN_MBEDTLS` is not enabled. Is this intentional?

Reply:
We still need `otError MbedTls::MapError(int rval)` even if external mbedTLS is used. `mbedtls.cpp` includes proper `#if` statements to exclude what's not needed.